app/components/session-modal/SessjonModal.tsx

```diff
@@ -12,7 +13,7 @@ interface IProps {
 }
 
 export function SessjonModal(props: IProps) {
-  const { session } = useRouteLoaderData("routes/rapportering") as ISessionLoader;
+  const { session } = useRouteLoaderData("routes/rapportering") as SerializeFrom<typeof loader>;
```

Comment:
Å type opp `useRouteLoaderData` på denne måten her synes jeg er mer ryddig. Vi slipper å type opp nye interfacer og bruker det vi får tilbake fra loader direkte. 

I tillegg matcher vi pathen der vi henter data fra. Pathen til `useRouteLoaderData` og `loader` er likt.

Reply:
Det er forskjellige måter å gjøre det på https://github.com/remix-run/remix/discussions/5061. Men synes at den her er enkleste.

Skulle ønske at remix eller react-router tilbyr det out of the box, men per nå får vi `unknow` 🙁 https://github.com/remix-run/react-router/blob/0e4c524b329bf9641ccfc85da95bcde708bc56ca/packages/react-router/lib/hooks.tsx#L771